impeller/core/allocator.cc

```diff
@@ -54,13 +77,27 @@ std::shared_ptr<Texture> Allocator::CreateTexture(
     return nullptr;
   }
 
+  if (enable_render_target_texture_cache_ && !desc.ignore_cache &&
```

Comment:
The ignore cache setting is required for this allocator cache to not fight with the swapchain caching we do for the MSAA textures. However, if this patch is succesful then we can remove the swapchain caching of MSAA textures and just rely on this.

I will make that change in a separate patch so that it is easier to understand the benchmark results.

Reply:
Add a TODO?